pkg/services/resourcesFilterService.go

```diff
@@ -12,6 +14,38 @@ type RootObject struct {
 }
 
 func ShouldResourceBeValidated(admissionReviewReq *admission.AdmissionReview, rootObject RootObject) bool {
+	isResourcesSkipByConfig := configAllowedListsValidation(admissionReviewReq, rootObject)
```

Comment:
I'm not a fan of breaking this boolean check into a separate function, it just adds more indirection with no abstraction benefit